src/platform/plugins/shared/vis_types/timeseries/public/application/components/vis_types/markdown/vis.js

```diff
@@ -48,9 +53,8 @@ function MarkdownVisualization(props) {
         ...variables,
       }
     );
-
     const markdownClasses = classNames('kbnMarkdown__body', {
-      'kbnMarkdown__body--reversed': isBackgroundInverted(panelBackgroundColor),
+      'kbnMarkdown__body--reversed': isReversed,
     });
```

Comment:
the `markdownClasses` declaration can be moved into the markdown component; we already set the `kbnMarkdown__body` class within our component, also since we already receive the `isReversed` prop within the component, we can also set the `kbnMarkdown__body--reversed` class in there, what's more I don't think we actually need to explicitly pass it since our new styles actually have no reference to it, however we should confirm that this classname is not being arbitrarily target from elsewhere.

Reply:
Since we aren't too sure, I suppose we can keep it around, however I think we can should contain the application of the classname within the markdown component.